Modules/Forum/classes/class.ilForumTopic.php

```diff
@@ -40,8 +40,8 @@ class ilForumTopic
 	private $is_closed = 0;
 	
 	private $orderField = '';
-	
-	private $posts = array();
+
+	public $last_post;
```

Comment:
@nmatuschek Could you please create a(n) mutator/accessor for this? As long as we do not have typed class properties I am not a fan of **public** properties.